tests/arch/test.yaml

```
@@ -0,0 +1,13 @@
+---
+{{- $architecture := or .architecture "amd64"}}
+architecture: {{$architecture}}
```

```suggestion
architecture: amd64
```
just hardcode it as we only support one arch anyway ?

---

For all the arch specific questions: as it's written there is nothing preventing you from trying other architectures - be that arm, aarch or risc-v. Personally I haven't tested them and instead opted for being consistent with the Debian/debootstrap/apt codebase.

Can change that, but it seems very strange to single-out the Arch/pacstrap/pacman actions.

---

If we are to hard-code things, should we do the same for Debian/debootstrap/apt?

---

https://archlinux32.org/ started since 32bit was dropped from the official distro - supports i486, i686 and pentium4
https://archlinuxarm.org/ has been a thing for 10+ years - supports armv7 and armv8 ... in the past it had armv6 support IIRC

---

There is also WIP risc-v somewhere (sorry cannot find the link).

All of the above should just work with the PR, of course assuming debos/qemu runs on that platform.

---

right - thanks for the history. I thought arch was arm64 only.

So, the named architectures here should just build ootb we can just resolve this.